src/Builder/AddressBuilder.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+
+namespace Prometee\SyliusPayumMoneticoPlugin\Builder;
+
+
+use Sylius\Component\Core\Model\AddressInterface;
+use Webmozart\Assert\Assert;
+
+class AddressBuilder implements BuilderInterface
```

Comment:
Each builders should have his own Interface, because they have different arguments and if someone wants to decorate it is would be easier for devs to use the right interface